.github/actions/setup/action.yml

```diff
@@ -36,7 +36,7 @@ runs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ inputs.python-version }}
-        cache: poetry
+#        cache: poetry
     - name: Set Poetry environment
```

Comment:
This file here and the Mongolia file are removed. The diff shown here is incorrect